tests/test_mongodb_full_table_interruptible.py

```diff
@@ -181,6 +198,28 @@ def test_run(self):
         self.assertEqual(records_by_stream['simple_coll_2']['messages'][-2]['data']['_id'],
                          interrupted_state['bookmarks']['simple_db-simple_coll_2']['max_id_value'])
 
+        # verify we are not seeing any documents which were updated having id < interrupted id value
+        self.assertNotEqual(999, records_by_stream['simple_coll_1']['messages'][0]['data']['int_field'])
+        self.assertNotEqual(888, records_by_stream['simple_coll_2']['messages'][0]['data']['int_field'])
+
+        int_value = False
+        for x in records_by_stream['simple_coll_1']['messages'][:-1]:
+            # We are not considering the last element of this list because it does not have 'data'
+            if int(x['data']['int_field']) == 999:
+                int_value = True
+        self.assertEqual(False, int_value)
+
+        int_value2 = False
+        for x in records_by_stream['simple_coll_1']['messages'][:-1]:
+            if x['data']['int_field'] == 888:
+                int_value2 = True
+        self.assertEqual(False, int_value2)
```

Comment:
Can you comment with what exactly this is checking. I am having a hard time following it.

Reply:
I have added relevant comments for others to understand